app.py

```diff
@@ -449,17 +489,25 @@ def loadFonts():
         if filename.endswith(".ttf"):
             QtGui.QFontDatabase.addApplicationFont(os.path.join(fontDir, filename))
 
-# Some issues responses have indicated that certain exceptions may not be 
-# displayed when Qt crashes unless this excepthook redirection is used. 
-sys._excepthook = sys.excepthook 
+# Some issues' responses have indicated that certain exceptions may not be
+# displayed when Qt crashes unless this excepthook redirection is used.
```

Comment:
And we still don't see them all. Silent crashes are a problem I've always had with `PyQt`. They're always my fault, but a little info would be nice.